hosted/helpers/helper_common.go

```diff
@@ -197,14 +182,16 @@ func GetCommonMetadataLabels() map[string]string {
 	// we only use this information instead of the ginkgo.CurrentSpecReport().FullText() because of the 63 character limit
 	var filename string
 	// Because of the way Support Matrix suites are designed, filename is not loaded at first, so we need to ensure it is non-empty before sanitizing it
+	//E.g. line51_k8s_chart_support_provisioning_test
 	if specReport.FileName() != "" {
 		// Sanitize the filename to fit the label requirements for all the hosted providers
-		filename = strings.Split(specReport.FileName(), "hosted/")[1] // abstract the relative path
-		filename = strings.TrimSuffix(filename, ".go")                // `.` is not allowed
-		filename = strings.ReplaceAll(filename, "/", "-")             // `/` is not allowed
-		filename = strings.ToLower(filename)                          // string must be in lowercase
+		fileSplit := strings.Split(specReport.FileName(), "/") // abstract the filename
```

Comment:
This change was made to avoid char length errors. Max permitted limit is 63.
The label will now be something like: line51_k8s_chart_support_provisioning_test